### DISCUSSION TITLE
SearchResult containing 0 data products

### DISCUSSION THREAD

**Discussion by avivajpeyi** [lightkurve/lightkurve#1213] [Q&A] [answered]
**QS why do some TIC not have any lightcurve data available?**

I am trying to download all the TESS TIC lightcurve files. 

**However, I have gotten null `SearchResults` for numerous TICs.** 

For example, see the following code snippet
```python
import lightkurve as lk
import pandas as pd
import numpy as np

np.random.seed(42)

EXOFOP = "https://exofop.ipac.caltech.edu/tess/"
TIC_DATASOURCE = EXOFOP + "download_toi.php?sort=toi&output=csv"

def data_availible(tic):
    search = lk.search_lightcurve(
        target=f"TIC {tic}", mission="TESS", author="SPOC"
    )
    if len(search)>1:
        return True
    return False


db = pd.read_csv(TIC_DATASOURCE)[["TIC ID", "TOI"]]
db = db.sample(100)
db['data_present'] = [data_availible(tic) for tic in db['TIC ID']]
print(f"{sum(db.data_present)}/{len(db)} TIC have data availible")
```

```
44/100 TIC have data availible
```

When I check the entire TESS TIC list, I see that 3026/5725 have data available (i.e. ~2K TIC do not have data available). 

Checking the ExoFop page for some of these TIC with null `SearchResult` objects, it seems like they should have data? For example, [TIC 53498154.](https://exofop.ipac.caltech.edu/tess/target.php?id=53498154)

**Comment by orionlee** [comment, marked as answer]
In general, a TIC is not guaranteed to have lightcurve products available for a variety of reasons.

In the specific case you reference(a sample of TOIs), most (if not all) would have lightcurve products, but not necessarily from the standard SPOC pipelines. Many lightcurves are produced from [QLP](https://archive.stsci.edu/hlsp/qlp) pipeline.
 
If you simply want some lightcurve products for a TIC, you can relax the search by omitting `author`:
```python
    search = lk.search_lightcurve(target=f"TIC {tic}", mission="TESS")
```

If you do have a specific TIC and find no lightcurve products, and you have checked that the target has been observed by TESS (e.g., by using [WTV](https://heasarc.gsfc.nasa.gov/cgi-bin/tess/webtess/wtv.py)), you can get the raw data by using [`search_tesscut()`](https://docs.lightkurve.org/reference/api/lightkurve.search_tesscut.html) and construct lightcurves.

**Reply by avivajpeyi** [reply]
thanks!
